packages/rocketchat-i18n/i18n/assistifyHelpRequest.en.i18n.yml

```diff
@@ -4,6 +4,7 @@ Choose_experts: Choose experts
 Close_HelpRequest: Close chat
 Close_request_comment: You can add (famous) last words
 Close_request_warning: Please do close the request once you are finished answering the questions asked. This way, we can learn from what you have written.
+Deactivate_close_comment: Allow Request closing comment
```

Comment:
should this not be the other way round: 
`Deactivate_close_comment: Don't allow closing comments`